### PR TITLE
Improves comms and key mismatch prevention while using the `init` CLI

### LINKED ISSUE #3528: Prevent nodes to startup if there's a Ferveo key mismatch
Related to #3527

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
Feature

**Required reviews:** 
2

**What this does:**
- Refines the operator experience for creating new nodes
- Blocks new node initialization on hosts with pre-existing local private keys or configurations
- Blocks new node initialization if operator account has already published a ferveo public key
- Expands communications regarding seed phrase security and responsibility while using the `init` CLI
- Alters console presentation of seed phrase to discourage copy/paste confirmation
- Begin to deprecate support for multiple configurations on a single host
- Handle testing and mocking side-effects of these changes
- Prevent tests from writing to production filepaths

**Issues fixed/closed:**
Assists #3527 
Closes #3528 
Closes https://github.com/nucypher/sprints/issues/37

**Why it's needed:**
Assists in accidental/intentional violation of operator protocols with regard to ferveo key commitment.

**Notes for reviewers:**

[asciicast](https://asciinema.org/a/5fhYGdBxOAbiEkE4GV3L3LPxn)

### CODE REVIEW

**Review by theref — APPROVED** [review]

File: nucypher/cli/painting/help.py [hunk 1]
```diff
@@ -30,36 +31,28 @@ def paint_new_installation_help(emitter, new_configuration, filepath):
...
 
-    default_config_filepath = True
-    if new_configuration.default_filepath() != filepath:
-        default_config_filepath = False
-    emitter.message(f'Generated configuration file at {"default" if default_config_filepath else "non-default"} '
-                    f'filepath {filepath}', color='green')
-
-    # add hint about --config-file
-    if not default_config_filepath:
-        emitter.message(f'* NOTE: for a non-default configuration filepath use `--config-file "{filepath}"` '
```
Comment (derekpierre) [inline nucypher/cli/painting/help.py:59]: Should we still print this note about using `--config-file` for non-default config filepath?

**Review by derekpierre — COMMENTED** [review]
Love the video 😍 ! I was going to ask about testing out the `init` command - and the video provided clarification.

File: nucypher/cli/painting/help.py (hunk 1 above)
Comment (KPrasch) [reply nucypher/cli/painting/help.py:59]: IMO this message is largely unnoticed, low-impact, and only applies to undocumented advanced use cases.  That being said it's easy to add back if you think it's still useful.

Comment (derekpierre) [reply nucypher/cli/painting/help.py:59]: 👍 . I defer to you here

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**